src/coord/Coord.cc

```diff
@@ -68,14 +66,20 @@ CoordSystemMap const getCoordSystemMap() {
     return idMap;
 }
 
-/// Calculate angular distance between two points using the Haversine formula
-///
-/// Besides the differences between the two coordinates, we also input the
-/// cosine of the two declinations, so as to be thrifty with the use of trig functions.
-afwGeom::Angle haversine(afwGeom::Angle const dAlpha, /// Difference between RAs (or longitudes), alpha1-alpha2
-                         afwGeom::Angle const dDelta, /// Difference between Decs (or latitudes), delta1-delta2
-                         double const cosDelta1, /// Cosine of the first Dec, cos(delta1)
-                         double const cosDelta2 /// Cosine of the second Dec, cos(delta2)
+/** Calculate angular distance between two points using the Haversine formula
+ *
+ * Besides the differences between the two coordinates, we also input the
+ * cosine of the two declinations, so as to be thrifty with the use of trig functions.
+ *
+ * @param dAlpha Difference between RAs (or longitudes), alpha1-alpha2
+ * @param dDelta Difference between Decs (or latitudes), delta1-delta2
+ * @param cosDelta1 Cosine of the first Dec, cos(delta1)
+ * @param cosDelta2 Cosine of the second Dec, cos(delta2)
+ */
+afwGeom::Angle haversine(afwGeom::Angle const dAlpha,
+                         afwGeom::Angle const dDelta,
+                         double const cosDelta1,
+                         double const cosDelta2
```

Comment:
That would make the process of documenting code more complex than it needs to be, so I'd rather keep things consistent.

Currently, the [Doxygen settings](https://github.com/lsst/base/blob/master/doc/base.inc#L435) disallow documentation of anonymous namespace members. I think that's the right place to make decisions about what is/isn't included in the API documentation.